catalogs/catalog-lakehouse-iceberg/src/main/java/com/datastrato/gravitino/catalog/lakehouse/iceberg/web/IcebergObjectMapper.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2023 Datastrato Pvt Ltd.
+ * This software is licensed under the Apache License version 2.
+ */
+
+package com.datastrato.gravitino.catalog.lakehouse.iceberg.web;
+
+import com.fasterxml.jackson.annotation.JsonAutoDetect;
+import com.fasterxml.jackson.annotation.PropertyAccessor;
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.PropertyNamingStrategies;
+import org.apache.iceberg.rest.RESTSerializers;
+
+public class IcebergObjectMapper {
+  private static final ObjectMapper objectMapper = createIcebergObjectMapper();
+
+  private IcebergObjectMapper() {}
+
+  private static ObjectMapper createIcebergObjectMapper() {
+    ObjectMapper mapper = new ObjectMapper();
+    mapper.setVisibility(PropertyAccessor.FIELD, JsonAutoDetect.Visibility.ANY);
+    mapper.configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false);
+    mapper.setPropertyNamingStrategy(new PropertyNamingStrategies.KebabCaseStrategy());
+    RESTSerializers.registerAll(mapper);
+    return mapper;
```

Comment:
The previouse implement(`JsonUtils.objectMapper()`) get a global ObjectMapper and applies Iceberg REST properties to it, this's not a good design.  it may effect other ObjectMappers.